.goreleaser.yaml

```diff
@@ -0,0 +1,197 @@
+      - -w -s
+      - -linkmode=external
+      - -extldflags '-Wl,-z,muldefs -static -lm'
+    tags:
+      - netgo
+      - ledger
+      - muslc
+      - osusergo
+  - id: elysd-linux-arm64
+    main: ./cmd/elysd
+    binary: elysd
+    hooks:
+      pre:
+        - wget https://github.com/CosmWasm/wasmvm/releases/download/{{ .Env.COSMWASM_VERSION }}/libwasmvm_muslc.aarch64.a -O /usr/lib/aarch64-linux-gnu/libwasmvm_muslc.a
+    goos:
+      - linux
+    goarch:
+      - arm64
+    env:
+      - CC=aarch64-linux-gnu-gcc
+    flags:
+      - -mod=readonly
+      - -trimpath
+    ldflags:
+      - -X github.com/cosmos/cosmos-sdk/version.Name=elys 
+      - -X github.com/cosmos/cosmos-sdk/version.AppName=elysd 
+      - -X github.com/cosmos/cosmos-sdk/version.Version={{ .Version }} 
+      - -X github.com/cosmos/cosmos-sdk/version.Commit={{ .Commit }}
+      - -X github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger,muslc,osusergo
+      - -w -s
+      - -linkmode=external
+      - -extldflags '-Wl,-z,muldefs -static -lm'
+    tags:
+      - netgo
+      - ledger
+      - muslc
+      - osusergo
+universal_binaries:
+  - id: elysd-darwin-universal
+    ids:
+      - elysd-darwin-amd64
+      - elysd-darwin-arm64
+    replace: false
+archives:
+  - id: zipped
+    builds:
+      - elysd-darwin-universal
+      - elysd-linux-amd64
+      - elysd-linux-arm64
+      - elysd-darwin-amd64
+      - elysd-darwin-arm64
+    name_template: "{{.ProjectName}}-{{ .Version }}-{{ .Os }}-{{ .Arch }}"
+    format: tar.gz
+    files:
+      - none*
+  - id: binaries
+    builds:
+      - elysd-darwin-universal
+      - elysd-linux-amd64
+      - elysd-linux-arm64
+      - elysd-darwin-amd64
+      - elysd-darwin-arm64
+    name_template: "{{.ProjectName}}-{{ .Version }}-{{ .Os }}-{{ .Arch }}"
+    format: binary
+    files:
+      - none*
+checksum:
+  name_template: "sha256sum.txt"
+  algorithm: sha256
+# Docs: https://goreleaser.com/customization/changelog/
+changelog:
+  skip: true
+# Docs: https://goreleaser.com/customization/release/
+release:
+  github:
+    owner: elys-network
+    name: elys
+  replace_existing_draft: true
+  header: |
+    < DESCRIPTION OF RELEASE >
+
+    ## Changelog
+
+    See the full changelog [here](https://github.com/elys-network/elys/blob/v{{ .Version }}/CHANGELOG.md)
+
+    ## ⚡️ Binaries
+
+    Binaries for Linux and Darwin (amd64 and arm64) are available below.
+    Darwin users can also use the same universal binary `elysd-{{ .Version }}-darwin-all` for both amd64 and arm64.
+
+    #### 🔨 Build from source
+
+    If you prefer to build from source, you can use the following commands:
+
+    ````bash
+    git clone https://github.com/elys-network/elys
+    cd elys && git checkout v{{ .Version }}
+    make install
+    ````
+  name_template: "elys v{{.Version}} 🧪"
+  mode: replace
+  draft: true
+
+# Docs: https://goreleaser.com/customization/announce/
+# We could automatically announce the release in
+# - discord
```

Comment:
Do we need to keep this commented lines for future use?

Reply:
yep those are additional options we might want to enable to automatically annonce a new release in discord and other channels we use